src/open_inwoner/scss/components/Contactmomenten/Contactmomenten.scss

```diff
@@ -74,46 +74,21 @@
 }
 
 .contactmoment {
-  &__details {
-    // Set table width for word-break
-    table {
-      margin-top: 0;
-      overflow-x: auto;
-      table-layout: fixed;
-      width: 100%;
-      @media (min-width: 768px) {
-        width: var(--mobile-ms-width);
-      }
-    }
-    table th {
-      width: 100px;
-      white-space: nowrap;
-    }
-    table td {
-      width: 300px;
-      overflow: hidden;
-      word-break: break-word;
-      white-space: normal;
-
-      &.table__item--status {
-        .table__item--wrapper {
-          display: flex;
-          justify-content: space-between;
-        }
-      }
+  p {
+    color: var(--font-color-body);
+  }
 
-      .case-detail__link {
-        color: var(--color-primary);
-        text-decoration: none;
+  &__heading:first-of-type {
+    margin-top: 0;
+  }
 
-        &::after {
-          font-family: 'Material Icons';
-          content: 'arrow_forward';
-          font-size: 1em;
-          vertical-align: middle;
-        }
-      }
-    }
+  &__heading {
+    font-size: var(--font-size-body-large);
+  }
+
+  &__text {
+    padding-bottom: var(--spacing-large);
+    line-height: var(--font-line-height-body-medium);
```

Comment:
No new line-height variable needed here because we already have that, it's in "--font-line-height-body: 24px" so you can use that variable.